src/gui/optionsdialog.ui

```diff
@@ -122,8 +122,8 @@
             <rect>
              <x>0</x>
              <y>0</y>
-             <width>501</width>
-             <height>893</height>
+             <width>493</width>
```

Comment:
These should be reverted and others below.

Reply:
>These should be reverted and others below.

Done.